maintenance/openstack-upgrade.md

```diff
@@ -139,4 +139,7 @@ varies by Linux distribution.
 
 1. Remove any existing `calicoctl` instances and [install the new `calicoctl`](../getting-started/clis/calicoctl/install).
 
+1. If you were upgrading from a version of Calico prior to v3.14 and followed the pre-upgrade steps for host endpoints above, review traffic logs from the temporary policy,
```

Comment:
Er - this host endpoints thing - does it apply to openstack?

Reply:
c3dd0d7